scripts/mkcloud

```diff
@@ -1235,6 +1245,16 @@ function sanity_checks()
     if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
+
+    if [[ " ${steplist[*]} " == *" batch "* ]] ; then
+        need_scenario=1
+        if [[ -n $scenario ]] ; then
+            scenariofile="${mkcloud_dir}/scenarios/cloud$(getcloudver)/${scenario}.yaml"
```

Comment:
I've updated the commit to reflect your feedback and @dirkmueller's suggestion to expect the scenario files to be in version-specific subfolders.
Do you think `${mkcloud_dir}/scenarios/` is ok or should it better be `${mkcloud_dir}/../scenarios/` as those scenarios aren't really scripts?